packages/controllers/src/snaps/selectors.ts

```diff
@@ -0,0 +1,4 @@
+import { TruncatedSnap } from '@metamask/snap-utils';
+
+export const getRunnableSnaps = (snaps: TruncatedSnap[]) =>
```

Comment:
Not sure how that makes a difference, it's only called with a `TruncatedSnap[]`.

If we change it, at least use a full name for the generic parameter like `Type`, please.

Reply:
Not sure I understand the reason for this request either

Reply:
Since it's in `snaps/` it might possibly be reused.

```typescript
const snaps: Snap[] = /* ... */;
const snaps2 = getRunnableSnaps(snaps); //
//^? const snaps2: TruncatedSnap[];
```